pkg/common/config/types.go

```diff
@@ -94,11 +102,18 @@ type VirtualCenterConfig struct {
 	CAFile string `gcfg:"ca-file"`
 	// Thumbprint of the VCenter's certificate thumbprint
 	Thumbprint string `gcfg:"thumbprint"`
+	// Secret Holder (intentionally not exposed via the config) is a label identify which
+	// InformerManager holds the secret
+	SecretHolder string
```

Comment:
Thoughts on `SecretRef` instead since this is just a reference to a secret resource right?

Reply:
Doesn't matter what the name is since it's internal. Will use SecretRef.